apps/platform/src/components/AssociationsToolkit/static_datasets/dataSourcesAssoc.js

```diff
@@ -147,7 +147,7 @@ const dataSources = [
     aggregationId: "affected_pathway",
     weight: 1,
     isPrivate: false,
-    docsLink: "https://platform-docs.opentargets.org/evidence#project-score", // TODO: check this
+    docsLink: "https://platform-docs.opentargets.org/evidence#crispr-screens", // TODO: check this
```

Comment:
Thanks @HelenaCornu ! Can you please remove the `TODO:` comment?

Reply:
Done!

Reply:
thanks ✅